tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/ObjectTest.java

```diff
@@ -47,11 +47,14 @@ public class ObjectTest {
 
     private CompileResult checkInInitializerResult;
     private CompileResult checkFunctionReferencesResult;
+    private CompileResult checkModuleVariableReferencesResult;
```

Comment:
Isn't this only when used as default values and therefore, closures? Please update the file, test, etc. names accordingly.

Reply:
Updated the names of test functions